socrates/test/subscribers/addon_object_test.js

```diff
@@ -21,8 +21,11 @@ describe('Subscriber\'s Addon', () => {
   });
 
   it('can be filled from the UI (unpacks t-shirt size)', () => {
-    const addonDataUI = {homeAddress: 'homeOne', billingAddress: 'billingTwo', tShirtSize: ['XXXL', '']};
-    const addonData = {homeAddress: 'homeOne', billingAddress: 'billingTwo', tShirtSize: 'XXXL', remarks: undefined};
+    const addonDataUI = {homeAddress: 'homeOne', billingAddress: 'billingTwo', tShirtSize: ['XXXL', ''], pronoun: 'she/her/her'};
+    const addonData = {
+      homeAddress: 'homeOne', billingAddress: 'billingTwo', diet: undefined, tShirtSize: 'XXXL',
+      needsAssistance: undefined, canProvideAssistance: undefined, remarks: undefined, pronoun: 'she/her/her'
+    };
```

Comment:
Also, einen Test für den Daten-Roundtrip würde ich mir hier schon wünschen... Du kannst z. B. den Test, der in Zeile 16 beginnt, vervollständigen.